department.rb

```diff
@@ -0,0 +1,37 @@
+# require './employee'
+
+class Department
+attr_reader :name, :roster
+  def initialize(name)
+    @name = name
+    @roster = []
+    @employees = []
+  end
+
+  def add_employee(employee)
+    @roster << employee
+  end
+
+  def total_salary
+    total = 0
+    @roster.each do |employee|
+      total += employee.salary
+    end
+    return total
+  end
+
+  def department_raise(raise)
+    satisfactory_salary = 0
+    satisfactory_employee = []
```

Comment:
You should probably call this variable `satisfactory_employees` since there will be more than one employee in it.